### ISSUE TITLE
[TESTING] Quest 1: Lava Walking Spell/Word of Power

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#213]
A small quest, that needs to be tested for quality and completeness.

Start at the magic shop in Glenn

**Comment by MrKnitster** [comment]
This was a fun one. Quick, but clear.

<details> 
  <summary>Spoiler</summary>
  Good amount of effort to get the clue from the woman at the magic shop, and then hunt down her sister. I like that this is treated as a throw-away by the NPC. Other than trial and error, is there a way we would discovery its use?
</details>

**Comment by MrKnitster** [comment]
<details> 
  <summary>Spoiler</summary>
  The only place I could think of to quickly test this was the forge in Minoc. That is filled with fire fields, not lava, but I assumed they were coded the same for damage. Worked like a charm! I'll try again on lava once I encounter some. Other than the Isle of the Avatar, I am not aware of any lava on the overworld surface or in any towns.
</details>

**Comment by MrKnitster** [comment]
Yes, I immediately saw the change. Maybe also a written clue when it is used, such as, "you feel a strange chill" or something similar?

**Comment by cambragol** [comment]
Maybe...but are there any other instances where a spell gives a written description to it? Otherwise it might be too one-offish or outside of normal parameters.

**Comment by MrKnitster** [comment]
Good point. Although you get a message when using the Wheel and the Horn. So I don't know.

**Comment by cambragol** [comment]
I went through all the spells, and there is no descriptive feedback. Only for Items. So I think adding prose would be a little jarring.

If I had thought it though really hard, I could have had the word discovered in a location where leaving (after testing the spell) would have been easier if the player slipped through a fireplace. When slipping through the fireplace, the power of the spell could be noticed because of the lack of damage feedback. Tricky to set up perfectly though...

**Comment by cambragol** [comment]
Ooooo....a perfect unused location. I think I have a beautiful solution...

**Comment by cambragol** [comment]
<details> 
  <summary>Spoiler</summary>
I am thinking to utilize the unused Ankh hidden past the lava in Glenn. 

Divia, in Redmount, instead of telling you the word, will tell you she learned it from an ankh in her childhood. Learning the word led her into seeking out 'words of power', but the word itself is forgotten by her.
The ankh told her the words to save her from the lava after she got lost in the cave in her childhood. She said the word, and escaped the cave, but did not realise that the word had charmed her to be protected from the lava/fire.
So the player must return to Glenn, and ask the ankh of Divia, and it will tell the tale, and the word.

Seems like an absolutely perfect use of the unsed Ankh in Glenn.
</details>

**Comment by cambragol** [comment]
Then again...just forcing the player to discover the secret via trial and error is kindof fun...

**Comment by MrKnitster** [comment]
> Then again...just forcing the player to discover the secret via trial and error is kindof fun...

That would be extremely devious!

**Comment by MrKnitster** [comment]
<details> 
  <summary>Spoiler</summary>
  I love the ankh idea! It's a waste to have the ankh be a deadend; this is a great use of it. I also think it provides a believable and immersive backstory. Divia grew up in Glenn, learned the secret there, was prompted to continue her studies of reagentless magic that took her to Redmount. The player starts in Glenn with her sister, who still lives there, and comes full circle back to Glenn after a trip to Redmount. Very cool!

For such a simple, old-fashioned interface from the 80s, that's some deep content.
</details>

**Comment by cambragol** [comment]
I have added the aforementioned step to this quest. It will still require the player to notice the effect of the spell, and cast/use it when told. And throws in a story to the mix. So I think it is an improvement.

**Comment by cambragol** [comment]
This change/tweak will be in 14.0
